src/metadata_parser/__init__.py

```diff
@@ -1948,8 +1949,7 @@ def _run_in_session(_requests_session: requests.Session):
                     )
                 log.error("NotParsable | %s", self.url)
                 raise NotParsable(
-                    "NotParseable document detected! "
-                    "content-type:'[%s]" % content_type,
+                    "NotParseable document detected! content-type:'[%s]" % content_type,
```

Comment:
nice improvement.

Reply:
Haha, yeah... Linter made this :sweat_smile: 

(It run on every save. To avoid having a commit that does linter+ modification, I preferred to make two commits. But yes, in this string use f-string will be a better choice)